Gemfile.lock

```diff
@@ -136,7 +136,7 @@ GEM
       tins (~> 1.6)
     crack (0.4.3)
       safe_yaml (~> 1.0.0)
-    crass (1.0.4)
+    crass (1.0.6)
```

Comment:
Not sure if that needs an update; this should result in an unnecessary update wrt the Debian archive.

Reply:
cf: https://tracker.debian.org/pkg/ruby-crass

Reply:
cf: https://github.com/flavorjones/loofah/blob/2595827d3e61d852ddbe533bb267ecbd15654dc0/loofah.gemspec#L29